bld/src/proxyd/log.config

```diff
@@ -5,6 +5,7 @@
 strict init=true
 buffer min=1024
 buffer max=0
+file perms=644
```

Comment:
Then 644 makes more sense, then giving read permissions to others but not to the own group.